views/apps/show.hbs

```diff
@@ -154,6 +158,29 @@
 </div>
 
 <script>
+  // WebSocket for updating app health
+  (() => {
+    document.addEventListener('DOMContentLoaded', function() {
+      const output = document.getElementById('app-health');
+      const websocket = new WebSocket(
+        'ws://localhost:3000/app-health?appId={{app.id}}'
```

Comment:
Was just thinking about this. We may need to set some kind of constant that can be referenced for this URL since once they deploy it won't be `localhost:3000`.

Reply:
Good catch. We could also maybe use  `window.location.host` to make it dynamic.